### ISSUE TITLE
GEOS aborting with "Assertion failed" on OS X

### ISSUE THREAD

**Issue by yeesian** [yeesian/ArchGDAL.jl#47]
see https://github.com/Toblerity/Shapely/issues/553

**Comment by sgillies** [comment]
@yeesian why did you reference that shapely issue? Is there a similar issue for your project? How does it manifest itself? Do you have any updates?

**Comment by yeesian** [comment]
Yeah, I did run into it on the date I opened this issue, but didn't have time to dig into it back then before moving on to subsequent binaries in https://github.com/JuliaGeo/GDALBuilder/releases, which did not have this issue. I left this issue open thinking that I'd have time to dig into it later.

That was at a point when there was still a lot of churn in the GDAL binaries I was trying out (https://github.com/yeesian/ArchGDAL.jl/pull/48) and I wasn't disciplined enough to log what I was doing, and have not been able to reproduce it now.
